sdk/rust/oak_app_build/src/main.rs

```diff
@@ -299,6 +299,9 @@ async fn main() -> anyhow::Result<()> {
 
     let output_file = get_output_path(&manifest_dir, &manifest.name);
 
+    std::fs::create_dir_all(manifest_dir.join(OUTPUT_DIRECTORY))
```

Comment:
Could this be moved into `get_output_path`? Since `get_output_path` is only called here.

Reply:
Done.